src/app/modules/expense/components/expense-mileage-form/expense-mileage-form.component.html

```diff
@@ -99,6 +105,33 @@ <h4 class="mileage-allowance-form__block__title">Indemnités kilométriques</h4>
                         </div>
                     </div>
                 </div>
+                <div *ngIf="vehiclesService.isCustomizable(this.commute)" [ngClass]="{
+                  'mileage-allowance-form__block__input-wrapper__item form-group':!vehiclesService.isCustomizable(this.commute),
+                  'mileage-allowance-form__block__input-wrapper__item2 form-group2':vehiclesService.isCustomizable(this.commute)
```

Comment:
Let the BEM class in all case, and add something like `3col`, `custom-rate` or something else with a `ng-class` is clearer no ?

Reply:
Or you can try something like that:
```
 class="mileage-allowance-form__block__input-wrapper__item" [ngClass]="{ 'mileage-allowance-form__block__input-wrapper__item--3col': vehiclesService.isCustomizable(this.commute) }"
```

and in the css you can use the BEM like before, and add this: 
```
&__item {
   &--3col {
   }
}
```

I'm not sure it will work without any modification, but I think it's the clearer :)